parity/run.rs

```diff
@@ -77,7 +77,13 @@ const SNAPSHOT_HISTORY: u64 = 100;
 const GAS_CORPUS_EXPIRATION_MINUTES: u64 = 60 * 6;
 
 // Pops along with error messages when a password is missing or invalid.
-const VERIFY_PASSWORD_HINT: &'static str = "Make sure valid password is present in files passed using `--password` or in the configuration file.";
+const VERIFY_PASSWORD_HINT: &str = "Make sure valid password is present in files passed using `--password` or in the configuration file.";
+
+// Full client number of DNS threads
+const FETCH_FULL_NUM_DNS_THREADS: usize = 4;
```

Comment:
I doubt we actually need all 4 for the full node either. But fine with keeping it the same for backward compatibility.